### PR TITLE
Refactor linkDetails a bit

### LINKED ISSUE #391: Unable to share portal
I've discovered that I'm not able to share some portals, while others work just fine.

I've found a couple and developed a theory since I found a thing they had in common. The ones that don't work contain the special letter "å" or "Å" (in caps). The letter is used in the Danish language, others are "æ/Æ" and "ø/Ø", however those work just fine. 

When I try to share the portal location, using the normal button from the portal info screen, nothing happens. I can see iitc register me clicking on the button, and normally there would open a menu on how or where to I would share that portal. Well with these, nothing happens. I can click the share button again, I've tried to wait and give it a minutes, but nothing. If I go back to the map and click on another portal, try and share that, it works as normal. If I then go back to the first portal again, still nothing.

This is one of those I've found, and containing the special letter which I suspect might be causing the problem, or at least have some influence on the problem:

https://intel.ingress.com/intel?ll=55.3366,10.331674&z=16&pll=55.33672,10.330928

### PR DESCRIPTION
**Pull request by johnd0e**
Fix #391

### CODE REVIEW

File: core/code/portal_detail_display.js [hunk 1]
```diff
@@ -78,27 +78,30 @@ window.renderPortalDetails = function(guid) {
...
-    var shareLink = $('<div>').html( $('<a>').attr({onclick:posOnClick}).text('Share portal') ).html();
-    linkDetails.push('<aside>'+shareLink+'</aside>');
+    var shareLink = $('<a>').text('Share portal').click(posOnClick);
+    linkDetails.append($('<aside>').append($('<div>').append(shareLink)));
 
   } else {
     // non-android - a permalink for the portal
-    var permaHtml = $('<div>').html( $('<a>').attr({href:permalinkUrl, title:'Create a URL link to this portal'}).text('Portal link') ).html();
-    linkDetails.push ( '<aside>'+permaHtml+'</aside>' );
+    var permaHtml = $('<a>').attr({
```
Comment (McBen) [inline core/code/portal_detail_display.js:94]: Note: you can even skip the "attr" here, like:
`$("<a>", {href: ...; title: "..."; text: "..."; click: ...})`

Comment (johnd0e) [reply core/code/portal_detail_display.js:94]: > Note: you can even skip the "attr" here, like: [...]

Yeah, but (imho) in that case readability would be worse.

Comment (McBen) [reply core/code/portal_detail_display.js:94]:
```suggestion
    var permaHtml = $('<a>', {
             text: 'Portal link',
             title: 'Create a URL link to this portal'},
             href: window.makePermalink([lat,lng])
         });
```

maybe

Comment (johnd0e) [reply core/code/portal_detail_display.js:94]: Yeah, may be. Matter of taste.

**Review by modos189 — APPROVED** [review]

**Pull request merged**